src/Middleware/Rewrite/src/IISUrlRewriteOptionsExtensions.cs

```diff
@@ -21,6 +22,7 @@ public static class IISUrlRewriteOptionsExtensions
         /// <param name="fileProvider">The <see cref="IFileProvider"/> </param>
         /// <param name="filePath">The path to the file containing UrlRewrite rules.</param>
         /// <param name="alwaysUseManagedServerVariables">Server variables are by default sourced from the server if it supports the <see cref="IServerVariablesFeature"/> feature. Use <c>true</c> to disable that behavior</param>
+        [SuppressMessage("ApiDesign", "RS0026:Do not add multiple public overloads with optional parameters", Justification = "<Pending>")]
```

Comment:
Either here or in a follow up, please fill in the `<Pending>` bit

Reply:
Whoops - good catch.